### DISCUSSION TITLE
[Feature] Hash update on metadata refresh

### DISCUSSION THREAD

**Discussion by waypones** [immich-app/immich#4137] [Feature Request]
### The feature

Currently metadata refresh (either for individual images or via "extract metadata" job) causes file hashes to get out of sync as they are not updated. Image hash should be also updated to keep database consistency. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by fr567** [comment]
Greetings @all,
is this metadata refresh in the meantime done, when the metadata is changed e.g. in Immich itself via app or web or manually on the original files in the file system? And if it is not done => Can I update the metadata in the file system without a problem? My use case is to bulk correct all the wrong dates from my Whatsapp images, which are uploaded via the Immich App.

**Reply by michaIII** [reply]
I would also like to use this scenario and am interested in the current status.
